src/behaviors.rs

```diff
@@ -13,10 +13,9 @@ pub enum RemoveBehavior {
     /// Use this behavior with caution.
     ///
     /// ```
-    /// use id_tree::Tree;
-    /// use id_tree::Node;
-    /// use id_tree::RemoveBehavior;
-    ///
+    /// # use id_tree::Tree;
+    /// # use id_tree::Node;
+    /// # use id_tree::RemoveBehavior;
     /// let mut tree: Tree<i32> = Tree::new();
```

Comment:
@Drakulix what do you think about me commenting out these `use` statements?  I thought they were a bit repetitive and just adding more noise to the doc tests.